packages/modules/src/modules/banners/utils/storybook.ts

```diff
@@ -42,6 +42,19 @@ export const content = {
     },
 };
 
+export const contentCharityAppeal = {
```

Comment:
We should move this variable to the banner story file and the `paragraphs` prop is missing in the options here. We put in both `messageText` and `paragraphs` as I believe there are plans to deprecate the former in the near future